app/models/target.rb

```diff
@@ -0,0 +1,7 @@
+class Target < ApplicationRecord
+  validates :topic, presence: true
```

Comment:
you can write this in 1 single line

` validates :topic, :lenght..., presence: true`